### DISCUSSION TITLE
2.9.1.0: setup executable never finishes

### DISCUSSION THREAD

**Discussion by emendelson** [The-Wineskin-Project/WineskinServer#246] [General]
Using 2.9.1.0 and WS11WineCX64bit21.1.0, and creating a new blank wrapper, in Monterey on Intel. When I try to run 64-bit executables as setup programs, I get the "Wineskin is currently busy" message, but the setup program never finishes. Possibly this is related to other problem reports about the latest version?

**Comment by Gcenx** [comment]
Yeah looks like I’d missed updating some strings sigh…

**Reply by emendelson** [reply]
OK - I'll stop experimenting!

**Reply by Gcenx** [reply]
Took a quick look and yep I’d missed a couple of strings within Wineskin.app

Quickly tested and it seemed to be working for me, downgraded MoltenVK while I was at it since 1.1.9 is super broken

**Reply by emendelson** [reply]
Yes, works perfectly now. Thank you!

**Comment by emendelson** [comment]
Just an FYI: Probably this is by design, but if I update an existing Wineskin app by updating the wrapper from 2.9.0.9-4 to 2.9.1.1., the app won't run. Nothing happens - no error message, just nothing. If I try a Test run, nothing happens, and if I ask to see the log, nothing appears. 

If this is by design, it may be worth warning users somewhere. (I know to test this kind of thing only when I have a backup of the app that I'm updating.)

**Reply by Gcenx** [reply]
With the structure change wineprefix from `/Contents/Resources` to `/Contents/ShareSupport/prefix` having Wineskin.app moving the files might cause problems.

Let's see if I remember to add manual steps for updating a wrapper to the wiki.
